main.go

```diff
@@ -84,11 +90,11 @@ func main() {
 	for {
 		select {
 		case <-signalCh:
-			log.Println("Shutdown signal received, stopping controllers...")
+			log.Printf("%s Shutdown signal received, stopping controllers...", logPrefix)
```

Comment:
should we update all Printf to corresponding levels like Infof / Warnf / Debugf ?